frontend/public/components/dashboard/details-card/detail-item.tsx

```diff
@@ -14,6 +14,6 @@ export const DetailItem: React.FC<DetailItemProps> = React.memo(({ title, value,
 
 type DetailItemProps = {
   title: string;
-  value?: string;
+  value?: any;
```

Comment:
the is PR https://github.com/openshift/console/pull/2520 which will change `DetailItem` in a way, that in will accept `children` instead of `value` which will support your case too and there will be no need to change this to `any`.